datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/types/dashboard/mappers/DashboardMapper.java

```diff
@@ -79,6 +81,14 @@ public Dashboard apply(@Nonnull final EntityResponse entityResponse) {
                 result.setInstitutionalMemory(InstitutionalMemoryMapper.map(new InstitutionalMemory(data)));
             } else if (GLOSSARY_TERMS_ASPECT_NAME.equals(name)) {
                 result.setGlossaryTerms(GlossaryTermsMapper.map(new GlossaryTerms(data)));
+            } else if (DOMAINS_ASPECT_NAME.equals(name)) {
+                final Domains domains = new Domains(data);
+                // Currently we only take the first domain if it exists.
```

Comment:
do we foresee this causing confusion?

Reply:
Did the same for containers.. I think it causes less confusion when you see it as a document model:

dataset {
    ....
    domains: [ ] 
}